handlers/workspaces_test.go

```diff
@@ -1200,7 +1201,83 @@ func TestGetWorkspaceUsers(t *testing.T) {
 }
 
 func TestGetUserDropdownWorkspaces(t *testing.T) {
+	teardownSuite := SetupSuite(t)
+	defer teardownSuite(t)
+
+	db.TestDB.DeleteWorkSpaceAllData()
+
+	oHandler := NewWorkspaceHandler(db.TestDB)
+	oHandler.userHasAccess = db.TestDB.DeleteWorkSpaceUserAccessData
```

Comment:
@MuhammadUmer44 This defeats the purpose, the function returns true it does not check if the user has access.